activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb

```diff
@@ -1,6 +1,7 @@
 require 'active_record/connection_adapters/abstract_mysql_adapter'
+require 'active_record/connection_adapters/mysql/database_statements'
 
-gem 'mysql2', '>= 0.3.18', '< 0.5'
+gem 'mysql2', '~> 0.4.4'
```

Comment:
Do we need to bump mysql2 minimum version? 0.4.x is only required if `prepared_statements: true`.